docs/.vitepress/theme/components/icons/SidebarIconCustomizer.vue

```diff
@@ -47,6 +47,10 @@ function resetStyle () {
   color.value = STYLE_DEFAULTS.color
   strokeWidth.value = STYLE_DEFAULTS.strokeWidth
   size.value = STYLE_DEFAULTS.size
+  absoluteStrokeWidth.value = STYLE_DEFAULTS.absoluteStrokeWidth
+  nextTick(() => {
+    
+  });
```

Comment:
Shouldn't this be inside the nextTick function?
```suggestion
  nextTick(() => {
    absoluteStrokeWidth.value = STYLE_DEFAULTS.absoluteStrokeWidth
  });
```